### PR TITLE
Feature/dashboard ui

### PR DESCRIPTION
**Pull request by gdmeridabit**

### CODE REVIEW

File: frontend_vue/src/components/units/UnitDetail.vue [hunk 1]
```diff
@@ -0,0 +1,222 @@
...
+            });
+            this.data = sample;
+          }
+      //   });
+    },
+    methods: {
+      onNext() {
+        //TODO: remove カード（normal） as initial value of learn_type
+        let question_id = 1;
+        let learn_type = 'カード（normal）';
```
Comment (hugh-n) [inline frontend_vue/src/components/units/UnitDetail.vue]: Name has been changed. New value is 'カード（FJ）';

File: frontend_vue/src/routes.js [hunk 2]
```diff
@@ -47,7 +47,7 @@ const routes = [
     }
   },
   {
-    path: '/unit/:id/test',
+    path: '/unit/:unit_id/test/:question_id',
```
Comment (hugh-n) [inline frontend_vue/src/routes.js]: Note that question doesn't exist anymore. Unless the backend is still sending data as a question?

File: frontend_vue/src/routing_util.js [hunk 3]
```diff
@@ -2,7 +2,12 @@ import router from './router';
 
 const routeUtil = {
   routeTo(learn_type, params) {
-    if(learn_type === 'カード') {
+    if(learn_type === 'カード（聴解）') {
```
Comment (hugh-n) [inline frontend_vue/src/routing_util.js]: カード（聴解）has been removed. Is this カード（JJ）( previously カード（QA) )?

File: frontend_vue/src/components/units/UnitDetail.vue (hunk 1 above)
Comment (gdmeridabit) [reply frontend_vue/src/components/units/UnitDetail.vue]: Is it カード（FJ）and カード（JJ）?

File: frontend_vue/src/routes.js (hunk 2 above)
Comment (gdmeridabit) [reply frontend_vue/src/routes.js]: This is still based on the old API, still waiting for the new

File: frontend_vue/src/components/units/UnitDetail.vue (hunk 1 above)
Comment (hugh-n) [reply frontend_vue/src/components/units/UnitDetail.vue]: Yeah

Comment (hugh-n) [reply frontend_vue/src/components/units/UnitDetail.vue]: I'd like to note the parenthesis is different. Should be カード(FJ) and カード(JJ).

Comment (gdmeridabit) [reply frontend_vue/src/components/units/UnitDetail.vue]: Done

File: frontend_vue/src/routing_util.js (hunk 3 above)
Comment (gdmeridabit) [reply frontend_vue/src/routing_util.js]: Done

File: frontend_vue/src/routes.js (hunk 2 above)
Comment (gdmeridabit) [reply frontend_vue/src/routes.js]: For this, I will just wait for the new implementation in the back end before I change this.

**Pull request merged**